backend/api/Controllers/InspectionController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Globalization;
+using Api.Controllers.Models;
+using Api.Database.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers
+{
+    [ApiController]
+    [Route("inspection")]
+    public class InspectionController(
+            ILogger<InspectionController> logger,
+            IInspectionService inspectionService
+        ) : ControllerBase
+    {
+
+        /// <summary>
+        /// Get the inspection image against an isarTaskId
```

Comment:
Not a fan of using "Against" in this description

```suggestion
        /// Lookup the inspection image for task with specified isarTaskId
```